pkg/profiler/profiler.go

```diff
@@ -215,36 +276,36 @@ func (p *CgroupProfiler) Run(ctx context.Context) error {
 	if err != nil {
 		return fmt.Errorf("get stack traces map: %w", err)
 	}
+	p.bpfMaps = &bpfMaps{counts, stackTraces}
```

Comment:
can we use field names here? I was confused and had to look at the `bpfMaps` type to understand what's going on here